test/index.test.js

```diff
@@ -80,6 +82,28 @@ describe('Sentry', function() {
         analytics.assert.deepEqual(window.RavenConfig.config, config);
       });
 
+      it('should allow and set custom versions', function() {
+        var config = {
+          logger: options.logger,
+          serverName: options.serverName,
+          whitelistUrls: options.whitelistUrls,
+          ignoreErrors: options.ignoreErrors,
+          ignoreUrls: options.ignoreUrls,
+          includePaths: options.includePaths,
+          maxMessageLength: options.maxMessageLength,
+          release: '2.4.0'
+        };
+
+        window.my_custom_version_property = '2.4.0';
+        analytics.initialize();
+
+        // Need to delete before asserts to prevent leaking effects in case of failure.
+        delete window.my_custom_version_property;
+
+        analytics.assert(window.RavenConfig.dsn === options.config);
```

Comment:
This was a copy of another test. I would guess that they were verifying that the same object was being used (thus no need to use deepEqual, since it would just be comparing the object to itself).